pkg/toolchain/label_value_test.go

```diff
@@ -0,0 +1,25 @@
+package toolchain_test
+
+import (
+	"testing"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/toolchain"
+
+	"gotest.tools/assert"
+)
+
+func TestLabelValue(t *testing.T) {
+	t.Run("string to valid value", func(t *testing.T) {
```

Comment:
you don't need a subtest here, but that's a minor thing

Reply:
removed: c53864a466a584bebc06050b27b064c0d94df582